boards/opentitan/earlgrey-cw310/.cargo/config.toml

```diff
@@ -2,5 +2,19 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 # Copyright Tock Contributors 2022.
 
+include = [
+  "../../../cargo/tock_flags.toml",
+  "../../../cargo/unstable_flags.toml",
+  "../../../cargo/riscv_flags.toml",
+  "../../../cargo/virtual_function_elimination.toml",
+  "../../../cargo/panic_abort_tests.toml",
```

Comment:
It would be nice to leave a comment about _why_ these are needed for earlgrey but not other boards.

Reply:
I fear the answer might be "I don't know. The Makefile already did that for some reason."